docs/usage/configuration-options.md

```diff
@@ -1268,10 +1268,12 @@ For example to apply a special label for Major updates:
 }
 ```
 
+Update type `patch` only matches if `separateMinorPatch` is set to true.
+
 ## patch
 
 Add to this object if you wish to define rules that apply only to patch updates.
-See also `major` and `minor` configuration options.
+Only applies if `separateMinorPatch` is set to true.
```

Comment:
You don't need to update `definition.ts` _and_ `configuration-options.md`. The docs that get deployed are a combination of both. You can see by taking a look at the [patch](https://docs.renovatebot.com/configuration-options/#patch) section online, which reads:

> Configuration to apply when an update type is patch. Only applies if separateMinorPatch is set to true